### PR TITLE
Add Delivery Hero to OpenTelemetry adopters

### PR DESCRIPTION
**Pull request by ekovalenk0**
Delivery Hero uses OpenTelemetry collectors and SDKs since at least a year (some parts of the company longer), hence it makes sense to add us to the list of OpenTelemetry adopters ;)

### CODE REVIEW

File: data/ecosystem/adopters.yaml [hunk 1]
```diff
@@ -20,6 +20,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: Delivery Hero
+  url: https://www.deliveryhero.com/
+  components: [Collector, Go, Java]
+  reference: ''
```
Comment (svrnm) [inline data/ecosystem/adopters.yaml:26]: Would you have a reference like a talk or blog post we can link out to? This helps potential adopters to learn from your experience

**Review by svrnm — CHANGES_REQUESTED** [review]
Thanks.

File: data/ecosystem/adopters.yaml [hunk 2]
```diff
@@ -20,6 +20,11 @@
...
+  contact: 'https://github.com/ekovalenk0'
```
Comment (chalin) [inline data/ecosystem/adopters.yaml:27]: Btw, here and elsewhere, we don't need quotes around URLs.
```suggestion
  contact: https://github.com/ekovalenk0
```

File: data/ecosystem/adopters.yaml (hunk 1 above)
Comment (ekovalenk0) [reply data/ecosystem/adopters.yaml:26]: There is no blogpost. There was a talk at the OTel meetup in Berlin but AFAIK it was not recorded. Once there is any material available, I will link it here ;)

**Review by svrnm — APPROVED** [review]

**Pull request merged**